### PR TITLE
Schema updates

### PR DESCRIPTION
**Pull request by jimduff-usds**
This PR does a few schema tweaks

## Changes
- Bizarrely, the HHS standard for patient_age_units is different from FHIR.  So I created a new valueset for this (hhs_age_units) but only using it for HHSProtect and for senders transitioning from Waters.
- specifically added ordering_facility_state to all direct schemas.
- removed weird transport type: NULL that was causing HL7_NULL tests to fail.
- forced date formats for HHSProtect to be very simple HHS standard.

### Testing
- [X] Tested locally?
- [X] Ran `quickTest all`?
- [X] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from `http://localhost:7071/api/download`?
- [ ] Added tests?

### Specific Security-related subjects a reviewer should pay specific attention to

- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Fixes
*List GitHub issues this PR fixes*
- #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

File: prime-router/metadata/schemas/WATERS/waters-covid-19.schema [hunk 1]
```diff
@@ -280,7 +280,7 @@ elements:
   # Time that waters recieved the record
   - name: waters_receive_date
     type: DATETIME
-    csvFields: [{ name: TXNTIMESTAMP}]
+    csvFields: [{ name: TXNTIMESTAMP, format: yyyyMMddhhmmss}]
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/WATERS/waters-covid-19.schema:283]: This actually didn't fix the problem Oracle team was having.  I created a ticket to fix our DATETIME formatting.

File: prime-router/metadata/schemas/direct/direct-covid-19.schema [hunk 2]
```diff
@@ -85,6 +85,15 @@ elements:
   - name: patient_age
     csvFields: [{ name: patientAge}]
 
+  # Using the HHS guidance instead of the FHIR UCUM codes (see https://www.hl7.org/fhir/valueset-age-units.html)
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/direct/direct-covid-19.schema:88]: Is this a hack?     HHS says to use 'yr' for year!  FHIR says to use, 'a'.  Where did THAT come from?

File: prime-router/metadata/schemas/direct/direct-covid-19.schema [hunk 3]
```diff
@@ -85,6 +85,15 @@ elements:
...
+  # In particular, using 'yr' for year instead of 'a'
+  - name: patient_age_units
+    type: CODE
+    pii: false
+    cardinality: ZERO_OR_ONE
+    valueSet: hhs_age_units
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/direct/direct-covid-19.schema:94]: New valueset.  See below.

File: prime-router/settings/organizations-local.yml [hunk 4]
```diff
@@ -1304,8 +1304,6 @@
         translation:
           type: HL7
           useBatchHeaders: true
-        transport:
```
Comment (jimduff-usds) [inline prime-router/settings/organizations-local.yml:1307]: These are the offending lines.

**Review by carlosfelix2 — APPROVED** [review]
Tested for the HL7_NULL and quality test failures.  Deleted the DB multiple times and switched between master and this branch.  Tests pass in this branch.

**Pull request merged**